app/controllers/admin/republishing_controller.rb

```diff
@@ -11,13 +11,13 @@ def republish_past_prime_ministers_index
     redirect_to(admin_republishing_index_path)
   end
 
-  def confirm
-    republishable_document = republishable_documents.find { |document| document[:slug] == params[:document_slug] }
+  def confirm_commonly_republished_document
```

Comment:
Hmm, is there a shorter name we can use here to distinguish between republishing a document with a slug and one without? I'm not sure if these are any more commonly-republished than the other type, and  I was under the impression neither of these were relatively commonly-republished! To help figure out the name for this type of document, what was the name you'd planned for the other type?

Reply:
Probably just `confirm_document`! I wonder if it's worth asking the others why these ones have specific Rake tasks - that might reveal a name?